packages/react-composites/tests/browser/call/CallComposite.test.ts

```diff
@@ -122,6 +122,16 @@ test.describe('Call Composite E2E CallPage Tests', () => {
     }
   });
 
+  test('local camera switcher button cycles camera', async ({ pages }, testInfo) => {
+    // mobile check
+    const testName = testInfo.project.name.toLowerCase();
+    test.skip(testName.includes('desktop'));
```

Comment:
* Please add a helper `isMobileView()` or such to contain this check, also add a comment why this string comparison works. (i.e., who sets the 'desktop' in the name).
* It would be better to create a new test suite for just the mobile tests. And if possible skip all tests at the suite level.

Reply:
okay I will make it so!